include/swift/SILOptimizer/Utils/Differentiation.h

```diff
@@ -17,12 +17,12 @@
 // NOTE: Although the AD feature is developed as part of the Swift for
 // TensorFlow project, it is completely independent from TensorFlow support.
 //
-// TODO: Move definitions here from Differentiation.cpp.
+// TODO: Move definitions from lib/SILOptimizer/Mandatory/Differentiation.cpp.
```

Comment:
I'm curious about what this TODO comment means. Do you plan to do sth like move definitions to a new file `lib/SILOptimizer/Utils/Differentiation.cpp`?

Reply:
This comment was already there. So, I just left it there. I presume that this is related to moving some helper functions out of Differentiation.cpp? @rxwei might have more context.

Reply:
Oh, I understand now - the comment existed previously in `lib/SILOptimizer/Mandatory/Differentiation.h`.